src/main/java/edu/harvard/hul/ois/fits/tools/jpylyzer/Jpylyzer.java

```diff
@@ -0,0 +1,141 @@
+    private static final Path UNIX_ROOT = TOOL_ROOT.resolve("unix");
+    private static final Path WINDOWS_ROOT = TOOL_ROOT.resolve("windows");
+    private static final String XSLT_FILE = Fits.FITS_XML_DIR + TOOL_NAME + File.separator + "jpylyzer_to_fits.xslt";
+
+    private final Fits fits;
+    private boolean enabled = true;
+    private final Path workingDir;
+    private final List<String> command;
+
+    public Jpylyzer(Fits fits) throws FitsException {
+        super();
+        log.debug("Initializing jpylyzer");
+
+        this.fits = fits;
+        info = new ToolInfo();
+        info.setName(TOOL_NAME);
+
+        String osName = System.getProperty("os.name");
+
+        if (osName.startsWith("Windows")) {
+            workingDir = WINDOWS_ROOT;
+            command = List.of("jpylyzer.exe");
+            info.setNote("jpylyzer for windows");
+            log.debug("jpylyzer will use Windows environment");
+        } else if (isPythonAvailable()) {
+            workingDir = UNIX_ROOT;
+            command = List.of("python", "-m", "jpylyzer");
+            info.setNote("jpylyzer for unix");
+            log.debug("jpylyzer will use Unix Python environment");
+        } else {
+            log.error("Python and Windows not supported, not running jpylyzer");
+            throw new FitsToolException("jpylyzer cannot be used on this system");
+        }
+
+        List<String> infoCommand = new ArrayList<>(command);
+        infoCommand.add("-v");
+        String versionOutput =
+                CommandLine.exec(infoCommand, workingDir.toAbsolutePath().toString());
+        info.setVersion(versionOutput.trim());
+    }
+
+    private static boolean isPythonAvailable() {
+        try {
+            String output = CommandLine.exec(List.of("python", "--version"), null);
+            if (output.startsWith("Python")) {
+                log.info("Found: {}", output);
+                return true;
+            }
+            return false;
+        } catch (Exception e) {
+            log.debug("Failed to execute to detect python version", e);
+            return false;
+        }
+    }
+
+    @Override
+    public ToolOutput extractInfo(File file) throws FitsToolException {
+        log.debug("jpylyzer starting on {}", file.getName());
+        long start = System.currentTimeMillis();
+
+        List<String> execCommand = new ArrayList<>(command);
+        execCommand.add(file.getAbsolutePath());
+
+        log.debug("Launching jpylyzer, command = {}", execCommand);
+        String execOut =
+                CommandLine.exec(execCommand, workingDir.toAbsolutePath().toString(), false);
+        log.debug("Finished running jpylyzer");
+
+        Document rawOut = parse(execOut);
+        Document fitsXml = transform(XSLT_FILE, rawOut);
+
+        output = new ToolOutput(this, fitsXml, rawOut, fits);
+
+        duration = System.currentTimeMillis() - start;
+        runStatus = RunStatus.SUCCESSFUL;
```

Comment:
Is `runStatus` used anywhere?

Reply:
Yeah, those fields are used by the super class.